libs/widget-lib/src/types.ts

```diff
@@ -199,10 +199,17 @@ export interface CowSwapWidgetParams {
    * Network ID.
    */
   chainId?: SupportedChainId
```

Comment:
Should we be more explicit here and call it `sourceChainId`? 

Although I like source/target notation, in the quote param I called it `sellTokenChainId`, because there's `sellTokenAddress` and others, so they follow the same naming convention. 

https://github.com/cowprotocol/cow-sdk/blob/bridging-sdk-getQuote/src/bridging/types.ts#L15

What do you think we should?

Reply:
We can rename it, but it will be a breaking change which will break existing integrations.
The only way is to add `sourceChainId` and keep `chainId`.